native/go/Makefile

```diff
@@ -19,8 +19,10 @@ include ../../mk/spksrc.native-cc.mk
 
 myComp:
 	@$(MSG) "Building Go"
-	cd $(WORK_DIR)/$(PKG_NAME)/src && ./make.bash
+	cd $(WORK_DIR)/$(PKG_NAME)/src && GOROOT_BOOTSTRAP=$(WORK_DIR)/../../go-1.4/work-native/go ./make.bash
```

Comment:
From a DRY pov: Can you define `GOROOT_BOOTSTRAP=$(WORK_DIR)/../../go-1.4/work-native/go` on L17 or thereabouts, then set `GOROOT_BOOTSTRAP=$(GOROOT_BOOTSTRAP)` instead?